### ISSUE TITLE
[BUG]  Unlabeled buttons in Sexual misconduct flow

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5414]
**Describe the bug**
In the  'Inappropriate Sexual behavior section of the app, there are a few items that are unlabeled for assistive technology such as screen readers. These need to have labels to convey their meaning to this user base. 

**To Reproduce
Steps to reproduce the behavior:**

Go to 'Inappropriate Sexual behavior section of the app

**Expected behavior**
The following items are labeled for assistive technology:
- The hamburger menu button to the right of "A Path Forward" 
- The information icon on Resource directory
- The quick exit icon
- The external link icon next to "National Domestic Violence hotline

**Actual behavior**
The following items are not labeled for assistive technology:

- The hamburger menu button to the right of "A Path Forward" 
- The information icon on Resource directory
- The quick exit icon
- The external link icon next to "National Domestic Violence hotline

Screenshots

![Image](https://github.com/user-attachments/assets/b9df1277-8a1e-49f9-b1f0-4a075194cb2a)

**Smartphone (please complete the following information):**
•	OS: Android Version 15
•	Device: Google Pixel 8a
•	Browser: Chrome 136.0.7103.87

**Additional context**
